<!DOCTYPE HTML>
<html>
	<head>
		<title>Publications - Winston Chan Lab</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
                                <link rel="icon" href="images/logo.ico"  type="image/x-icon">
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Header -->
					<header id="header">
						<a href="index.html" class="logo">Winston Lab</a>
					</header>

				<!-- Nav -->
					<nav id="nav">
						<ul class="links">
							<li><a href="index.html">Home</a></li>
							<li><a href="research.html">Research</a></li>
							<li class="active"><a href="publications.html">Publications</a></li>
<li><a href="members.html">Members
</a></li>
<li><a href="join.html">Join us
</a></li>
<li><a href="contact.html">Contact us
</a></li>
						</ul>
						
					</nav>

				<!-- Main -->
					<div id="main">

						<!-- Post -->
							<section class="post">
								<header class="major">
									<h1>Selected Publications</h1>
								</header>

								<!-- Text stuff -->
<header>
										<h2>A Decade of Liver Organoids: Advances In Disease Modeling</h2>
<p>Yue Liu, Jian-Ying Sheng, Chun-Fang Yang, Junjun Ding and Yun-Shen Chan (2023) Clinical and Molecular Hepatology.  </p>
</header>
<h3>Study abstract</h3>
									<p><span class="image left"><img src="images/Publications1.jpg" alt="" /></span>Liver organoids are three-dimensional cellular tissue models in which cells interact to form unique structures in culture. During the past 10 years, liver organoids with various cellular compositions, structural features, and functional properties have been described. Methods to create these advanced human cell models range from simple tissue culture techniques to complex bioengineering approaches. Liver organoid culture platforms have been used in various research fields, from modeling liver diseases to regenerative therapy. This review discusses how liver organoids are used to model disease, including hereditary liver diseases, primary liver cancer, viral hepatitis, and nonalcoholic fatty liver disease. Specifically, we focus on studies that used either of two widely adopted approaches: differentiation from pluripotent stem cells or epithelial organoids cultured from patient tissues. These approaches have enabled the generation of advanced human liver models and, more importantly, the establishment of patient-tailored models for evaluating disease phenotypes and therapeutic responses at the individual level.
									<hr />
<header>
										<h2>Advancements in MAFLD Modeling with Human Cell and Organoid Models</h2>
<p>Shi-Xiang Wang, Ji-Song Yan, Yun-Shen Chan (2022) International Journal of Molecular Sciences.   </p>
</header>
<h3>Study abstract</h3>
									<p><span class="image right"><img src="images/Publications2.jpg" alt="" /></span>Metabolic (dysfunction) associated fatty liver disease (MAFLD) is one of the most prevalent liver diseases and has no approved therapeutics. The high failure rates witnessed in late-phase MAFLD drug trials reflect the complexity of the disease, and how the disease develops and progresses remains to be fully understood. In vitro, human disease models play a pivotal role in mechanistic studies to unravel novel disease drivers and in drug testing studies to evaluate human-specific responses. This review focuses on MAFLD disease modeling using human cell and organoid models. The spectrum of patient-derived primary cells and immortalized cell lines employed to model various liver parenchymal and non-parenchymal cell types essential for MAFLD development and progression is discussed. Diverse forms of cell culture platforms utilized to recapitulate tissue-level pathophysiology in different stages of the disease are also reviewed.
									<hr />
<header>
<h2>Krϋppel-like factor 5 rewires NANOG regulatory network to activate human naive pluripotency specificLTR7Ys and promote naive pluripotency</h2>
<p> Zhipeng Ai, Xinyu Xiang, Yangquan Xiang, Iwona Szczerbinska, Yuli Qian, Xiao Xu, Chenyang Ma, Yaqi Su, Bing Gao, Hao Shen, Muhammad Nadzim Bin Ramli, Di Chen, Yue Liu, Jia-jie Hao, Huck Hui Ng, Dan Zhang, Yun-Shen Chan#, Wanlu Liu#, Hongqing Liang# (2020) Cell Reports. 
</p>										
									</header>
<h3>Study highlights</h3>
<ul><span class="image left"><img src="images/publications3.jpg" alt="" /></span>
												<li>Human naive pluripotency enriched LTR7Ys are activated by KLF4 and KLF5</li>
												<li>KLF4 and KLF5 have both overlapping and distinct gene and TE targets</li>
												<li>KLF5 rewires NANOG binding affinity from LTR7s to LTR7Ys</li>
												<li>KLF5-NANOG activates LTR7Ys to boost naive gene expression and expands cell potency</li>
											</ul>
<hr />
								
<header>
<h2>Human Pluripotent Stem Cell-Derived Organoids as Models of Liver Diseases</h2>
<p>Muhammad Nadzim Bin RAMLI, Yee Siang LIM, Hongqing LIANG, Kevin Andrew Uy GONZALES, Weiquan TNG, Cheng Peow TAN, Iwona SZCZERBINSKA, Chwee Tat KOE, Lewis LU, Ka Man YU, Lai Ping YAW, John Soon Yew LIM, Graham WRIGHT, Asim SHABBIR, Yock Young DAN, Huck-Hui NG, Yun-Shen CHAN (2020) Gastroenterology.  
</p>

										
</header>
<h3>Study highlights</h3)
<ul><span class="image right"><img src="images/pic09.jpg" alt="" /></span>
												<li>A step wise, chemically defined and high-throughput approach for efficient generation of hepatic organoids from pluripotent stem cells</li>
												<li>The liver organoids recapitulate the structural and functional aspects of the bile canaliculi system</li>
												<li>The bile canaliculi system in the liver organoid can be employed to study drug induced cholestasis</li>
												<li>The liver organoids enable the first demonstration of fatty liver induced ductular reaction in vitro </li>
											</ul>
<hr />
<header>
<h2>A Chemically Defined Feeder-free System for the Establishment and Maintenance of the Human Naive Pluripotent State</h2>
<p>Szczerbinska I, Gonzales KAU, Cukuroglu E, Ramli N, Tan CP, Göke J, Ng HH, and Chan YS. (2019) Stem Cell Report</p>

										
									</header>
<ul><span class="image left"><img src="images/pic08.jpg" alt="" /></span><h3>Study highlights</h3>

												<li>High-throughput screen to identify compounds liberating naïve hESCs from feeder dependency</li>
												<li>Establishment and maintenance of human feeder-independent naive embryonic (FINE) stem cells</li>
												<li>FINE cells are molecularly equivalent to 4iLA hESCs</li>
												<li>FINE cells offer enhanced genomic stability and amenability to introduction of exogenous DNA</li>

											</ul>
<hr />	
<header>
										
										<h2>Induction of a Human Pluripotent State with Distinct Regulatory Circuitry that Resembles Preimplantation Epiblast</h2>
<span class="image right"><img src="images/pic07.jpg" alt="" /></span><p>Chan YS, Göke J, Ng JH, Lu X, Gonzales KA, Tan CP, Tng WQ, Hong ZZ, Lim YS, Ng HH. (2013) Cell Stem Cell. </p>

										
									</header>
<h3>Study highlights</h3>
<ul>
												<li>A combination of three small molecules induces a LIF-dependent hESC state (3iL)</li>
												<li>3iL hESCs share an expression signature with native preimplantation epiblast</li>
												<li>The transcriptional regulatory circuitry is remodeled in 3iL hESCs</li>
												<li>Identification of enhancers of human preimplantation epiblast genes</li>
											</ul>
<hr />
<header>
										<h2>Genome-wide kinase-chromatin interactions reveal the regulatory network of ERK signaling in human embryonic stem cells
</h2>
<p>Göke J, Chan YS, Yan J, Vingron M, Ng HH. (2013) Molecular Cell</p>

										
									</header>
<h3>Study highlights</h3>
<ul><span class="image left"><img src="images/pic06.jpg" alt="" /></span>
												<li>Genome-wide kinase-chromatin interactions for ERK2 in hESCs.</li>
												<li>ERK2 targets noncoding, histone, cell-cycle, and metabolism genes.</li>
												<li>Motif finding and colocalization analysis reveal candidate partners of ERK2.</li>
												<li>ELK1 colocalizes with ERK2 and Polycomb and is linked to pluripotency in hESCs.</li>
											</ul>
	<hr />
<header>
										<h2>A genome-wide RNAi screen reveals determinants of human embryonic stem cell identity</h2>
<p>Chia NY*, Chan YS*, Feng B*, Lu X, Orlov YL, Moreau D, Kumar P, Yang L, Jiang J, Lau MS, Huss M, Soh BS, Kraus P, Li P, Lufkin T, Lim B, Clarke ND, Bard F, Ng HH. (2010) Nature </p>

										
									</header>
<h3>Study abstract</h3>
									<p><span class="image right"><img src="images/pic05.jpg" alt="" /></span>The derivation of human ES cells (hESCs) from human blastocysts represents one of the milestones in stem cell biology. The full potential of hESCs in research and clinical applications requires a detailed understanding of the genetic network that governs the unique properties of hESCs. Here, we report a genome-wide RNA interference screen to identify geneswhich regulate self-renewal and pluripotency properties in hESCs. Interestingly, functionally distinct complexes involved in transcriptional regulation and chromatin remodelling are among the factors identified in the screen. To understand the roles of these potential regulators of hESCs, we studied transcription factor PRDM14 to gain new insights into its functional roles in the regulation of pluripotency. We showed that PRDM14 regulates directly the expression of key pluripotency gene POU5F1 through its proximal enhancer. Genome-wide location profiling experiments revealed that PRDM14 colocalized extensively with other key transcription factors such as OCT4, NANOG and SOX2, indicating that PRDM14 is integrated into the core transcriptional regulatory network. More importantly, in a gain of-function assay, we showed that PRDM14 is able to enhance the efficiency of reprogramming of human fibroblasts in conjunction with OCT4, SOX2 and KLF4. Altogether, our study uncovers a wealth of novel hESC regulators wherein PRDM14 exemplifies a key transcription factor required for the maintenance of hESC identity and the reacquisition of pluripotency in human somatic cells.
</p>
									<hr />

<header>
										<h2>A core Klf circuitry regulates self-renewal of embryonic stem cells
</h2>
<p>Jiang J, Chan YS, Loh YH, Cai J, Tong GQ, Lim CA, Robson P, Zhong S, Ng HH. (2008) Nature Cell Biology. </p>

										
									</header>
<h3>Study abstract</h3>
									<p><span class="image left"><img src="images/pic04.jpg" alt="" /></span>Embryonic stem (ES) cells are unique in their ability to self-renew indefinitely and maintain pluripotency. These properties require transcription factors that specify the gene expression programme of ES cells. It has been possible to reverse the highly differentiated state of somatic cells back to a pluripotent state with a combination of four transcription factors: Klf4 is one of the reprogramming factors required, in conjunction with Oct4, Sox2 and c-Myc. Maintenance of self-renewal and pluripotency of ES cells requires Oct4, Sox2 and c-Myc, but Klf4 is dispensable. Here, we show that Krüppel-like factors are required for the self-renewal of ES cells. Simultaneous depletion of Klf2, Klf4 and Klf5 lead to ES cell differentiation. Chromatin immunoprecipitation coupled to microarray assay reveals that these Klf proteins share many common targets of Nanog, suggesting a close functional relationship between these factors. Expression analysis after triple RNA interference (RNAi) of the Klfs shows that they regulate key pluripotency genes, such as Nanog. Taken together, our study provides new insight into how the core Klf circuitry integrates into the Nanog transcriptional network to specify gene expression that is unique to ES cells.
</p>
	
							</section>

					</div>

				<!-- Footer -->
					<footer id="footer">
						<section class="split contact">
							<section class="alt">
								<h3>Email</h3>
								<p>chan_yunshen@gzlab.ac.cn，wu_liyan2@gzlab.ac.cn</p>
							</section>
							
						</section>
					</footer>

				<!-- Copyright -->
					<div id="copyright">
						<ul><li>&copy; Winston Lab</li><li>Guangzhou,China</li></ul>
<p>This website is open-source and available on Github.It uses template from <a href="https://html5up.net">HTML5 UP</a></p>					
</div>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
